test/library/keypress_and_caret.spec.js

```diff
@@ -342,6 +342,24 @@ describe('Test keypress and caret position changes', () => {
       jasmine.clock().uninstall()
     });
 
+    it('should clear active timers', () => {
+      jasmine.clock().install();
+
+      const wrapper = mount(<NumberFormat />);
+      const instance = wrapper.instance();
+
+      simulateFocusEvent(wrapper.find('input'), 0, 0, setSelectionRange);
+
+      expect(instance.focusTimeout).toBeTruthy();
+
+      wrapper.unmount();
+      jasmine.clock().tick(1);
+
+      expect(instance.focusTimeout).toBeFalsy();
+
+      jasmine.clock().uninstall();
```

Comment:
If the spec fails, other test will fail, as the clock will still be mocked by this spec. 
You should clear mock on afterEach. Do unInstall there.